src/prometheus-stack/chart/templates/uds-package.yaml

```diff
@@ -7,7 +7,43 @@ metadata:
   name: prometheus-stack
   namespace: {{ .Release.Namespace }}
 spec:
+  {{- if .Values.sso.enabled }}
+  sso:
+    - name: uds-prometheus
+      clientId: uds-prometheus
+      redirectUris:
+        - "https://prom.admin.{{ .Values.domain }}/auth"
+      enableAuthserviceSelector:
+        app.kubernetes.io/name: prometheus
+      groups:
+        anyOf:
+          - /UDS Core/Admin
+          - /UDS Core/Auditor
+    - name: uds-alertmanager
+      clientId: uds-alertmanager
+      redirectUris:
+        - "https://alerts.admin.{{ .Values.domain }}/auth"
+      enableAuthserviceSelector:
+        app.kubernetes.io/name: alertmanager
+      groups:
+        anyOf:
+          - /UDS Core/Admin
+          - /UDS Core/Auditor
+  {{- end }}
   network:
+    expose:
+      - service: alertmanager-operated
+        selector:
+          app.kubernetes.io/name: alertmanager
+        host: alerts
+        gateway: admin
+        port: 9093
+      - service: prometheus-operated
+        selector:
+          app: prometheus
+        host: prom
```

Comment:
open to other names here 🤷

Reply:
Currently we have grafana and neuvector, might suggest we keep with that pattern and just use the full product name here.

Reply:
(alternatively we could try and lean more into functionality based naming like `sso` is, so `alerts` and `metrics`?)

Reply:
I like `alerts` and `metrics`.  It just provides a better UX.  Switched to that